metadata-ingestion/src/datahub/api/entities/corpgroup/corpgroup.py

```diff
@@ -81,10 +81,4 @@ def emit(
         :param callback: The callback method for KafkaEmitter if it is used
         """
         for mcp in self.generate_mcp():
-            if type(emitter).__name__ == "DatahubKafkaEmitter":
-                assert callback is not None
-                kafka_emitter = cast("DatahubKafkaEmitter", emitter)
-                kafka_emitter.emit(mcp, callback)
-            else:
-                rest_emitter = cast("DatahubRestEmitter", emitter)
-                rest_emitter.emit(mcp)
+            emitter.emit(mcp, callback)
```

Comment:
Previously callback was mandatory for Kafka Emitter, or at least it failed with an assertion if it was not set. 
Is it fine if we lose this?

Reply:
what is callback used in the Kafka emitter?

Reply:
we've changed it so the callback is not mandatory for kafka

by default, the kafka emitter will log error messages